tests/components/ChatInput.test.tsx

```diff
@@ -205,3 +205,12 @@ it("executes custom handleError if provided", async () => {
   expect(customHandleError).toBeCalledTimes(1);
   expect(customHandleError).toBeCalledWith("API Error");
 });
+
+it("executes onSend if provided", async () => {
+  console.log = jest.fn();
```

Comment:
nit: instead of overriding console.log, lets make a separate function 
```
onSendCb = jest.fn();
render(<ChatInput onSend={onSendCb} ...} />)
```
and then check if onSendCb is called